src/app/containers/MediaPlayer/index.jsx

```diff
@@ -63,11 +63,14 @@ const MediaPlayerContainer = ({
     aresMediaBlock,
   );
   const duration = moment.duration(rawDuration, 'seconds');
+  const durationSpokenPrefix = translations.media.duration
```

Comment:
```suggestion
  const durationSpokenPrefix = pathOr('Duration', ['media', 'duration'], translations);
```

Reply:
Ahh okay so this a fancier way of doing a ternary expression? 

```
const durationSpokenPrefix = translations.media.duration
    ? translations.media.duration
    : 'Duration';
```